app.js

```diff
@@ -106,7 +106,6 @@ global.Chat = require('./chat');
 
 global.Rooms = require('./rooms');
 
-delete process.send; // in case we're a child process
```

Comment:
That line's a holdover from when app.js was handling dependency installation. It's not anymore. Even if app.js were to be ran as a child process for some other reason, I don't think mutating `process` like that's the best way to go around it.